userguide/one_shot_object_detection/README.md

```diff
@@ -62,18 +61,60 @@ To learn more about deploying One-Shot to CoreML details, refer to the [Object D
 
-To begin training, a set of synthetic training data needs to be generated using the provided starter image(s). 
+To begin training, the provided starter image(s) are used to generate a set of synthetic training data. 
 The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images. If no background images are provided, a set of default background images are automatically downloaded and used.
 
 **Stage 2: Train the Object Detector model**
 
 To learn about the training implementation details, refer to the [Object Detector: How it Works](https://apple.github.io/turicreate/docs/userguide/object_detection/how-it-works.html) chapter.
 
+## Troubleshooting
+
+Here is a collection of some known caveats and their respective workarounds while using One-Shot Object Detection.
+
+##### My starter image can appear upside down as well as rotated 90 degrees in the wild. Is there anything I can do to improve the quality of the model?
+
+To ensure a higher quality model, instead of providing *one* starter image, feel free to provide *more than one* starter images. For every starter image, we rotate the starter image by up to 20 degrees. As a result, to detect both inverted and erect projections of the same starter image, you can provide *2* starter images with the same label. Here is an example to rotate a Stop Sign starter image by 90 degrees, 180 degrees, and 270 degrees, and construct an SFrame with four starter images:
+
+```python
+import turicreate as tc
+
+def pil_to_tc(pil_image):
+    pixel_data = np.array(pil_image)
+    FORMAT_RAW = 2
+    return tc.Image(_image_data=pixel_data.tobytes(),
+                    _width=pixel_data.shape[1],
+                    _height=pixel_data.shape[0],
+                    _channels=pixel_data.shape[2],
+                    _format_enum=FORMAT_RAW,
+                    _image_data_size=pixel_data.size)
+
+image_path = # set image path here
+tc_image = tc.Image(image_path)
+pil_image = Image.open(image_path)
+images = [tc_image]
+for rotation_angle in [90, 180, 270]:
+    rotated = pil_to_tc(pil_image.rotate(rotation_angle, expand=True))
+    images.append(rotated)
+
+starter_images = tc.SFrame({
+    'image': images,
+    'label': ['stop_sign', 'stop_sign', 'stop_sign', 'stop_sign']
+})
+```
 
+##### While training with the default backgrounds, my starter image does not appear in most of the images in the synthetic data.
+
+This is a known issue ([#2341](https://github.com/apple/turicreate/issues/2341)) that we're trying to fix. As a workaround, please try resizing your starter image (while preserving the aspect ratio of the starter image) to get it as close to `500x500` (for the default backgrounds) as possible. You can leverage the following API provided by `turicreate.Image`:
+
+```python
+image = turicreate.Image('starter.png')
+turicreate.image_analysis.resize(image, width=image.width/2, height=image.height/2, channels=image.channels)
```

Comment:
Is it cool if `image.width` or `image.height` is odd and we're not in python 2? In other words what happens if the values we pass to `resize` for `width` or `height` is a non-whole number float? Maybe change `/` -> '//'.